applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py

```diff
@@ -456,6 +456,10 @@ def test_SmallDisplacementElement_3D_hexa(self):
 
         #self.__post_process(mp)
 
+        # Testing explicit utilities
+        max_delta_time = StructuralMechanicsApplication.CalculateDeltaTime(mp, 2.0, 1.0e-3, 0.5)
+        self.assertAlmostEqual(max_delta_time,1.3234236058132611e-07)
```

Comment:
why so you add the test here?

Reply:
I am not going to create a new test for two lines of code...